File: 03/piglatin.py
def piglatin(str):
    str = str.lower()
    string = ''
    vowelc = is_vowel(str[0])
    if (vowelc):
        string = str+ 'ay'
        return string
    elif (vowelc == False):
        string = str[1:] + str[0] + 'ay'
        return string
        
        
def is_vowel(a):
    if (a == 'a' or a == 'e' or a=='i' or a=='o' or a=='u'):
        return True
    else:
        return False
    
print(piglatin('HIPPO'))
print(piglatin('aunt'))

File: 14/piglatin.cpp
#include <iostream>
#include <string>
#include <algorithm>

using std::cout;
using std::endl;
using std::string;

bool is_vowel(char a){
	if (a == 'a' || a == 'e' || a == 'i' || a == 'o' || a == 'u'){
		return true;
	}
	else {
		return false;
	}
}

std::string piglatin(std:: string str){
	int l = str.length();
	for (int i = 0; i < l; ++i)
	{
		str[i] = tolower(str[i]);
	}
	string s;
	if (is_vowel(str[0])){
		s = str + "ay";
		return s;
	}
	else{
		s = str.substr(1,l) + str[0] + "ay";
		return s;
	}
}

int main(){
	cout << piglatin("HIPPO") << endl;
	cout << piglatin("aunt") << endl;
}